code/modules/projectiles/guns/projectile/automatic/straylight.dm

```diff
@@ -13,15 +17,18 @@
 	mag_well = MAG_WELL_SMG
 	magazine_type = /obj/item/ammo_magazine/smg
 	auto_eject = 1
-	matter = list(MATERIAL_PLASTEEL = 16, MATERIAL_PLASTIC = 4)
-	price_tag = 2500 //good smg with normal recoil and silencer possibility
+	matter = list(MATERIAL_PLASTEEL = 7, MATERIAL_STEEL = 3, MATERIAL_PLASTIC = 10)
```

Comment:
I understand that 16 is a lot, but 7 is too small number.

Reply:
Adressed, changed to 12, 2, 8.